../../lib/berkeley-db-1.xx/include/berkeley-db/cdefs.h:58: error: "__P" redefined [-Werror]
   58 | #define __P(protos)     protos          /* full-blown ANSI C */
      | 

---

--- a/lib/berkeley-db-1.xx/include/berkeley-db/cdefs.h
+++ b/lib/berkeley-db-1.xx/include/berkeley-db/cdefs.h
@@ -55,7 +55,10 @@
  * strings produced by the __STRING macro, but this only works with ANSI C.
  */
 #if defined(__STDC__) || defined(__cplusplus)
+#ifdef	__P
+#undef	__P
 #define	__P(protos)	protos		/* full-blown ANSI C */
+#endif /* !__P */
 #define	__CONCAT(x,y)	x ## y
 #define	__STRING(x)	#x
 
